### PR TITLE
issue #522: CurrencyToken with EF issue

### PR DESCRIPTION
**Pull request by xuzhg**
issue #522: CurrencyToken with EF issue

### CODE REVIEW

File: OData/src/System.Web.OData/OData/Query/Expressions/SelectExpandBinder.cs [hunk 1]
```diff
@@ -298,8 +299,8 @@ private Expression ProjectElement(Expression source, SelectExpandClause selectEx
                 Dictionary<IEdmNavigationProperty, ExpandedNavigationSelectItem> propertiesToExpand = GetPropertiesToExpandInQuery(selectExpandClause);
                 ISet<IEdmStructuralProperty> autoSelectedProperties;
 
-                IEdmEntitySet entityset = _context.NavigationSource as IEdmEntitySet;
-                ISet<IEdmStructuralProperty> propertiesToInclude = GetPropertiesToIncludeInQuery(selectExpandClause, entityType, entityset, _model, out autoSelectedProperties);
+                // IEdmEntitySet entityset = _context.NavigationSource as IEdmEntitySet;
```
Comment (luoyan0517) [inline OData/src/System.Web.OData/OData/Query/Expressions/SelectExpandBinder.cs]: Remove this?

**Comment by luoyan0517** [comment]
Looks good to me.

File: OData/src/System.Web.OData/OData/Query/Expressions/SelectExpandBinder.cs (hunk 1 above)
Comment (xuzhg) [inline OData/src/System.Web.OData/OData/Query/Expressions/SelectExpandBinder.cs]: Good

**Comment by xuzhg** [comment]
Merged. Thanks.

File: OData/test/E2ETest/WebStack.QA.Test.OData/ETags/DominiosController.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Web.Http;
+using System.Web.OData;
+
+namespace WebStack.QA.Test.OData.ETags
+{
+    public class DominiosController : ODataController
+    {
+        private ETagCurrencyTokenEfContext _db = new ETagCurrencyTokenEfContext();
```
Comment (congysu) [inline OData/test/E2ETest/WebStack.QA.Test.OData/ETags/DominiosController.cs]: CurrencyToken -> ConcurrencyToken

**Pull request closed without merging**